### PR TITLE
Remove Azure pipeline, add Github Pipeline + Codecov

### PR DESCRIPTION
**Pull request by evberrypi**
Removes Azure Pipeline in favor of GitHub pipeline (since pipeline is CPU only)
Enables Codecov extension

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - name: Run Unit tests
+        run: |
+          source .venv/bin/activate
+          ./run test all unit
+      - name: Coverage
+        run: |
+          source .venv/bin/activate
+          coverage xml
+      - name: Upload coverage
+        uses: codecov/codecov-action@v1
```
Comment (gigony) [inline .github/workflows/pr.yml:40]: Please update this to v2 as we need to do anyway early next year when v1 is no longer available.

https://github.com/codecov/codecov-action

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        with:
+          fail_ci_if_error: false
+          file: ./coverage.xml
```
Comment (gigony) [inline .github/workflows/pr.yml:43]: nit: no ends with blank link

**Review by gigony — APPROVED** [review]
Looks good to me. Thanks Everett!
Approved with some comments.

File: .github/workflows/pr.yml (hunk 2 above)
Comment (evberrypi) [reply .github/workflows/pr.yml]: Updated

**Pull request merged**